test/integrations/routes/users.test.js

```diff
@@ -14,167 +13,157 @@ chai.use(chaiHttp);
 let dummyUser;
 
 describe('API endpoint: /api/users', () => {
-  before(() => {
-    // synchronize and create tables
-    models.sequelize.sync();
-  });
-  beforeEach(() => {
-    // reset the dummy user data
-    dummyUser = { ...DUMMY_USER };
-    // drop tables
-    Bluebird.all([
-      models.User.destroy({ truncate: true }),
-    ]);
-  });
+  describe('POST /api/v1/users', () => {
+    beforeEach(() => {
```

Comment:
#### Nitpick
- Why are you dropping a table in `beforeEach` and not dropping tables after the tests are all done.

Reply:
I am not dropping tables...the before each block is for a dummy user

Reply:
@fejiroofficial I see you're `truncating` rather